elytron/WFCORE-5145-ssl-allowed-protocols.adoc

```diff
@@ -0,0 +1,112 @@
+= [WFCORE-5145] Elytron server-ssl-context allowed protocols
+:author:            Sonia Zaldana Calles
+:email:             szaldana@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Elytron currently supports 6 SSL/TLS protocols. Namely, SSLv2, SSLv3, TLSv1, TLSv1.1, TLSv1.2, and
+TLSv1.3. However, older JDK versions use ``SSLv2Hello`` during the initial
+SSL handshake message where the SSL version that will be used for the rest of the handshake is negotiated.
+ This task is to ensure Elytron can make use of the ``SSLv2Hello`` protocol to ensure feature parity with legacy security.
```

Comment:
Would be good to mention here that newer JDK versions disable this protocol by default but do provide the ability to re-enable it if needed.

Reply:
Fixed